packages/element/README.md

```diff
@@ -199,6 +209,8 @@ _Returns_
 
 ### createRoot
 
+_Since: WordPress 6.2_
```

Comment:
Excellent, idea. The same annotation needs to get applied in JSDoc comment where this function gets exported.

Reply:
Not sure what you need me to do here?